packages/owl-bot/src/scan-and-retry-failed-lock-updates.ts

```diff
@@ -20,6 +20,8 @@ const twentyFourHours =
   60 * // minutes
   24; // hours.
 
+const seventyTwoHours = 3 * twentyFourHours;
```

Comment:
nit: I'd probably have named this constant what it's used for rather than the value with a comment describing the human readable value.